variables.tf

```diff
@@ -28,6 +28,31 @@ variable "instance_tenancy" {
   default     = "default"
 }
 
+variable "public_subnet_suffix" {
+  description = "Suffix to append to public subnets name"
+  default     = "public"
+}
+
+variable "private_subnet_suffix" {
+  description = "Suffix to append to private subnets name"
+  default     = "private"
+}
+
+variable "database_subnet_suffix" {
+  description = "Suffix to append to database subnets name"
+  default     = "database"
```

Comment:
This was `db` before

Reply:
done

Reply:
Sorry for the confusion :) I just discovered that there was an inconsistency in the code - it was a mix of `database` and `db`.